cloud/parameterstore/fakeparameter/parameter_manager_test.go

```diff
@@ -0,0 +1,236 @@
+package fakeparameter
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/evergreen/cloud/parameterstore"
+	"github.com/evergreen-ci/evergreen/db"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// Tests for the ParameterManager functionality are in this package to avoid a
+// circular dependency between the two packages.
+func TestParameterManager(t *testing.T) {
+	defer func() {
+		assert.NoError(t, db.ClearCollections(Collection))
```

Comment:
nit: require

Reply:
It doesn't really matter if it's assert or require in this defer since the test is already over by the time this runs. Mostly I just leave the assert/require to know if failing to clear the collection affects a later test.